### PR TITLE
update macros translate for zh-CN

### PR DESCRIPTION
**Pull request by mutoe**
- update outdated `commonly-used_macros` translation
- add `other` macros translate
- fix 'macros' index link

### CODE REVIEW

**Review by irvin — CHANGES_REQUESTED** [review]
this duplicates #2173 , please rebase after it get merged.

**Comment by irvin** [comment]
#2173 had been merged, please rebase your branch. (or you can close this one and open another new one instead.)

File: files/zh-cn/mdn/structures/macros/commonly-used_macros/index.html [hunk 1]
```diff
@@ -195,75 +188,73 @@ <h2 id="Status_and_compatibility_indicators">状态和兼容性指示器</h2>
...
 <h5 id="Examples_2"><code>示例</code></h5>
 
 <ul>
-  <li>图标：{{experimental_inline}}</li>
+  <li>图标：{{Experimental_Inline}}</li>
 </ul>
 
 <h3 id="Inline_indicators_that_support_specifying_the_technology">代表明确技术参考的行内指示器</h3>
 
-<h4 id="Deprecated">不赞成的</h4>
```
Comment (pocketdr) [inline files/zh-cn/mdn/structures/macros/commonly-used_macros/index.html:228]: https://github.com/mdn/content/pull/1047#issuecomment-902335443

> This should broadly be interpreted to mean "use of this API is discouraged"

File: files/zh-cn/mdn/structures/macros/commonly-used_macros/index.html [hunk 2]
```diff
@@ -142,27 +140,23 @@ <h3 id="Linking_to_bugs">关联到某个 Bug</h3>
...
+<p>{{TemplateLink("Previous")}}、{TemplateLink("Next")}} 和 {{TemplateLink("PreviousNext")}} 这几个宏可以在页面中创建导航栏，帮助读者按照文章的先后顺序阅读。其中的参数需要填入目标页面在 MDN 中的位置，你可以在页面的网址中找到所需的信息。对于 {{TemplateLink("PreviousNext")}}，需要的两个参数是相应文章的 Wiki 位置。第一个参数用于上一篇文章，第二个参数用于下一篇文章。</p>
 
 <h2 id="Code_samples">代码示例</h2>
 
 <h3 id="Live_samples">动态示例</h3>
 
 <ul>
-  <li>{{TemplateLink("EmbedLiveSample")}} 可嵌入一个在<a href="/zh-CN/docs/MDN/Contribute/Structures/Live_samples">动态示例</a>中描述的代码示例，到当前页面上。</li>
-  <li>{{TemplateLink("LiveSampleLink")}} 创建一个页面链接，其中包含如<a href="/zh-CN/docs/MDN/Contribute/Structures/Live_samples">动态示例</a>中描述的示例的结果。</li>
-  <li>{{TemplateLink("EmbedGHLiveSample")}} 提供了一种新的动态示例编写和使用方式，你可以在<a href="/zh-CN/docs/MDN/Structures/Code_examples#github_live_samples">Github动态示例</a>中了解更多信息。</li>
```
Comment (pocketdr) [inline files/zh-cn/mdn/structures/macros/commonly-used_macros/index.html:165]: 已经提交 PR 在原文里添加这一段，https://github.com/mdn/content/pull/8440#issue-721899817

Comment (mutoe) [reply files/zh-cn/mdn/structures/macros/commonly-used_macros/index.html:165]: added, 等待合并

**Comment by mutoe** [comment]
this PR is pending on https://github.com/mdn/content/pull/8440

File: files/zh-cn/mdn/structures/macros/commonly-used_macros/index.html (hunk 1 above)
Comment (pocketdr) [reply files/zh-cn/mdn/structures/macros/commonly-used_macros/index.html:228]: 这里还没有改~~

**Review by irvin — APPROVED** [review]

**Pull request merged**